ajax/libs/tiny-date-picker/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "tiny-date-picker",
+  "description": "A small, dependency-free date picker",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/chrisdavies/tiny-date-picker.git"
+  },
+  "keywords": [
+    "date",
+    "picker",
+    "date",
+    "chooser",
+    "calendar"
+  ],
+  "author": "Chris Davies <ferociousturtle@gmail.com>",
+  "license": "MIT",
+  "homepage": "https://github.com/chrisdavies/tiny-date-picker#readme",

```

Comment:
@ArmenAvetisyan We'll use git repo as homepage by default if the custom homepage is not found, so there is no need to set homepage to its git repo. Let's remove it.